plugins/importexport/native/filter/AuthorNativeXmlFilter.inc.php

```diff
@@ -22,6 +22,8 @@ class AuthorNativeXmlFilter extends PKPAuthorNativeXmlFilter {
 	 */
 	function __construct($filterGroup) {
 		parent::__construct($filterGroup);
+
+		$this->getNoValidation(true);
```

Comment:
(This is temporary pending tomorrow's work, I think?)

Reply:
Fixed